publiccode.yml

```diff
@@ -0,0 +1,62 @@
+publiccodeYmlVersion: '0.2'
+name: Nuovo OPAC di polo SBN
+releaseDate: '2019-02-04'
+url: 'https://github.com/IstitutoCentraleCatalogoUnicoBiblio/Nuovo-Opac-di-Polo-SBN'
+developmentStatus: stable
+softwareType: standalone/web
+platforms:
+  - web
+usedBy:
+  - >-
+    Istituto Centrale per il Catalogo Unico delle Biblioteche Italiane e per le informazioni bibliografiche
+categories:
+  - OPAC (On-Line-Public-Access-Catalogue)
```

Comment:
Purtroppo i valori di questa chiave sono da scegliere in una lista controllata presente qui: https://docs.italia.it/italia/developers-italia/publiccodeyml/it/master/categories-list.html#tag-validi
```suggestion
```

Reply:
Ok, modificato